src/go/k8s/apis/core/v1alpha1/webhook_suite_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2021 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package v1alpha1_test
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1alpha1 "github.com/vectorizedio/redpanda/src/go/k8s/apis/core/v1alpha1"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	// +kubebuilder:scaffold:imports
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+	"sigs.k8s.io/controller-runtime/pkg/envtest/printer"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+// These tests use Ginkgo (BDD-style Go testing framework). Refer to
```

Comment:
In the long run, we should decide if we want to follow the same patterns between all our Go projects ( namely operator & rpk) regarding testing frameworks.

Reply:
Agree. The kubebuilder pave the way for BDD-style go testing. Personally, I prefer standard go testing framework with `assert` and `require` helper library.

Reply:
:+1: that's exactly what we use for rpk.